### PR TITLE
Nuclear operative medical options

### PR DESCRIPTION
**Pull request by Birdtalon**
Replaces the random syringe gun and charcoal syringes on the nuclear operative shuttle with a bone repair kit and regular first aid kit (icon and name changed because nukies are ❄️). I have never seen the syringe gun used and the charcoal syringes seem rather an odd choice when the nukies face mainly burn/brute damage.

Also makes the syndicate tactical medical kit 3 TC cheaper (now 4TC) and removes the combat defib from it to replace with an extra synthflesh patch. Nukies tend to explode when they die, making it rather useless and 7TC is way too much.

Hopefully will encourage more use of teamwork and more field medics if people can get the NV medhud at a reasonable price.

dreammaker_2018-05-23_22-40-11

🆑 Birdtalon
tweak: Nukie shuttle now gets a bone repair kit and regular first aid kit instead of syringe gun and charcoal.
tweak: Syndicate medical kit made cheaper with defibrillator removed in place of an extra patch.
/🆑

### CODE REVIEW

**Comment by theColdflame** [comment]
Can the meth be touched on in some way here?

It's an incredibly potent addition to the ops' kit which they aren't made aware of in the slightest since the pockets are visually indistinct from normal ones. 

Trading it out for better free meds or making it more clear free stims are available would be nice.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -170,7 +170,7 @@
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
 	new /obj/item/reagent_containers/food/pill/patch/synthflesh(src)
-	new /obj/item/defibrillator/compact/combat/loaded(src)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/firstaid.dm]: This is kinda the biggest reason you buy the kit. I think people severely underestimate the strength of this thing--it's insane, yet no one touches it.

Comment (theColdflame) [reply code/game/objects/items/weapons/storage/firstaid.dm]: People shouldn't be buying medical supplies because they come with an instakill weapon

**Comment by EMPlight** [comment]
I've never been able to use the med kit because I though it was way to expensive. Giving Nuke Ops another cheaper healing option besides the medgun is great.

File: code/game/objects/items/weapons/storage/firstaid.dm (hunk 1 above)
Comment (Birdtalon) [reply code/game/objects/items/weapons/storage/firstaid.dm]: You so rarely see medics on nuke ops. Sometimes people will occasionally purchase the medbeam gun. I feel like the combat defib is relatively unknown and inflates the price of the medical supplies box substantially. I have never seen this medical kit be purchased and the NV medhud is pretty critical for a nukie medic. I think the 7 TC price tag puts people off, you could get an emag for that or nearly an esword.

I could potentially move the combat defib into a more specialised medic bundle with stronger items to reflect a greater cost, or just have it as a standalone item as the medgun is.

Comment (EMPlight) [reply code/game/objects/items/weapons/storage/firstaid.dm]: You could have two medkits, a 4TC personal basic syndi kit, and the 7TC advanced one.

**Comment by Dumbdumn5** [comment]
With the changes to the syndicate medkit in place, what do you think about the 7 TC price in its current state?

**Pull request closed without merging**